hack/istio/multicluster/kiali-prepare-remote-cluster.sh

```diff
@@ -244,6 +246,16 @@ create_kiali_remote_cluster_secret() {
     error "The remote cluster name [${REMOTE_CLUSTER_NAME}] does not conform to Kubernetes rules for secret key data. Use --remote-cluster-name to specify a name that matches the regex '^[-._a-zA-Z0-9]+$'"
   fi
 
+  if [ "${EXEC_AUTH_JSON}" != "" ]; then
+    local user_auth=$(cat <<EOF
+env:
+$(echo ${EXEC_AUTH_JSON} | yq -P | sed "s/^/  /g")
```

Comment:
Without wrapping in quotes, you get this:

`Error: bad file '-': yaml: mapping values are not allowed in this context`

To see this, create a `/tmp/exec-auth.yaml` with:

```
exec:
  apiVersion: client.authentication.k8s.io/v1beta1
  command: echo
  args:
  - |
    {
      "apiVersion": "client.authentication.k8s.io/v1beta1",
      "kind": "ExecCredential",
      "status": {
        "token": "token"
      }
    }
  interactiveMode: IfAvailable
```

Then run
```
hack/istio/multicluster/kiali-prepare-remote-cluster.sh -dr true -eaj "$(cat /tmp/exec-auth.yaml)"
```

To fix:

```suggestion
$(echo "${EXEC_AUTH_JSON}" | yq -P | sed "s/^/  /g")
```

Reply:
this fix is included in my [PR](https://github.com/frauniki/kiali/pull/1)